network-libp2p/src/discovery/peer_contacts.rs

```diff
@@ -299,35 +316,43 @@ impl PeerContactBook {
     /// If the filter matches the services provided by the contact, it is added.
     /// Otherwise it is ignored.
     /// The services_filter argument to this function contains the services that are required.
-    pub fn insert_filtered(&mut self, contact: SignedPeerContact, services_filter: Services) {
+    pub fn insert_filtered(
+        &mut self,
+        contact: SignedPeerContact,
+        services_filter: Services,
+        only_secure_connections: bool,
+    ) {
         let info = PeerContactInfo::from(contact);
 
-        if self
+        // A peer is interested to us in two cases:
```

Comment:
```suggestion
        // A peer is interesting to us in two cases:
```
Looks like grammar but inverts the meaning.